ubuntu/clang/uptodate.yaml

```diff
@@ -3,24 +3,24 @@ dockerbuild:
   # Specific combinations of build args to exclude (optional)
   # Spack doesn't work with 14 nor 15
   exclude:
-    llvm_version: ["14.0.0", "15.0.0"]
+    llvm_version: ["14", "15"]
     ubuntu_version: ["20.04", "20.04"]
 
   build_args:
     llvm_version:
       key: llvm
-      name: llvm
-      type: spack
-
-      # Only install x.0.0 versions
-      filter: 
-        - "^[0-9]+[.]0[.]0$"
-
-      # Greater than 10.0.0
-      startat: "10.0.0"
+      versions:
+        - "10"
+        - "11"
+        - "12"
+        - "13"
+        - "14"
+        - "15"
+        - "16"
```

Comment:
Do these versions work without adding the the test apt repository?

https://github.com/LLNL/axom-docker/blob/main/ubuntu-22/base/generic-setup.sh#L27-L28

Also I think the exclude lines above are not right. I specifically need 14 for sure.